Haskell/Blarney/Core/Net.hs

```diff
@@ -14,19 +14,17 @@ License     : MIT
 Maintainer  : mattfn@gmail.com
 Stability   : experimental
 
-This module provides types and functions to represent circuits as 'Netlist's
-that can then be rendered as Verilog or in other formats...
+This module provides types and functions to represent circuits as
```

Comment:
This no longer provides types. Let's rename the module to NetHelpers altogether?